### DISCUSSION TITLE
Welcome to MeshLab Discussions!

### DISCUSSION THREAD

**Discussion by alemuntoni** [cnr-isti-vclab/meshlab#832] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by SmartNan** [comment]
How to use the MeshLab, I am a green hand. Recently, I need to deal with some points colud, I heard that the MeshLab can do that, I download it, but I don't know how to use it, just know open points cloud data.
As you can see, my English is poor!

**Reply by alemuntoni** [reply]
Hi @SmartNan, happy you are starting to use MeshLab! :)
To get started, you can find a set of tutorial youtube videos on our website: https://www.meshlab.net/#support

**Reply by SmartNan** [reply]
Thank you! I am willing to watching the youtube videos!

**Reply by SmartNan** [reply]
If I have some problem, may I ask questions at here?

**Reply by alemuntoni** [reply]
Yes, you could ask here (opening a new discussion with Q&A category), or use stackoverflow with `meshlab` as a tag.
